book/plugins/android/single-bundle.md

```diff
@@ -3,35 +3,38 @@
 ![single-bundle.png](./single-bundle.png)
 
 ### OVERVIEW
-Single bundle is a plugin that makes multiple react native components implementation stable, increase performance and it is more efficient and easy to handle in a native app. This plugin gives ability to change version of react native and react. It also aggregate all react native plugins into one single bundle and upload it to S3. Plugins becomes modules under the names specified as plugin identifiers from plugin configuration.
+Single Bundle is a plugin that makes the implementation of multiple React Native components stable, it increases performance and it is more efficient and easy to handle in a native app. This plugin gives the ability to change the versions of React Native and React. It also aggregates all the React Native plugins into a single bundle and it uploads it to S3. Plugins become modules under the names specified as plugin identifiers from the plugin configuration.
 
-### PLUGINS PREPERATIONS
+### PLUGIN PREPERATION
+
+#### Add "Single Bundle Plugin" to the Project Configuration
```

Comment:
I know that it's just a language improvements P.R
But is it possible to add the relevant JSON part as an example?

Reply:
Do you mean the JSON extract from the Single Bundle manifest with the plug-in parameters? I'd rather make another pull request for this.